src/CarbonAware.WebApi/test/integrationTests/IntegrationTestingBase.cs

```diff
@@ -127,6 +127,15 @@ public void Setup()
     [SetUp]
     public void SetupTests()
     {
+        if (_dataSource == DataSourceType.JSON)
+        {
+            // To force WebApplication to consume new JSON datasorce it needs to be restarted.
```

Comment:
spelling nits: `datasource` and `CarbonAware`

Reply:
fixed